### ISSUE TITLE
Fix torch deprecation error

### ISSUE THREAD

**Issue by laxitshahi** [llm-proxy/llm-proxy#142]
When running LLM Proxy a deprecation error is shown. Locate the source of the deprecation and ensure no warning messages show up:

![Image](https://github.com/llm-proxy/llm-proxy/assets/46412618/4b5e386b-97d0-4610-97ff-5f0055d86068)

**Comment by dinuja9** [comment]
Its a fix that needs to happen within the transformer library, and so we have to wait on them to fix it. Ignore the warning for now.

**Comment by laxitshahi** [comment]
Downgraded torch to resolve error; Another task will be created to upgrade torch and transformers library in the future to ensure compatibility.
